infrastructure/kube/argocd/proxy.sh

```
@@ -0,0 +1 @@
+kubectl port-forward svc/argocd-server -n argocd 8080:443
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)